### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gulraiznoorbari**
Renamed the bash script and updated the bash script help section in README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,15 @@
-# conda-remove-env
+# conda_remove_env
 
 Remove one or more conda environments
 
-
 1. Clone the repo
 
 ```
-git clone https://github.com/trevor-moon/conda-remove-envs.git
```
Comment (trevor-moon) [inline README.md:9]: This line can remain unchanged - the link will be broken since the url uses the repo name `conda-remove-envs`

**Review by trevor-moon — COMMENTED** [review]
Please see inline comment

File: README.md (hunk 1 above)
Comment (gulraiznoorbari) [reply README.md:9]: PR Updated

**Comment by gulraiznoorbari** [comment]
PR Updated.

Also updated the script name (usage & examples section) in the bash script file

**Review by trevor-moon — APPROVED** [review]

**Pull request merged**